models/remote_provider.go

```diff
@@ -2064,6 +2064,7 @@ func (l *RemoteProvider) TokenHandler(w http.ResponseWriter, r *http.Request, fr
 	ck := &http.Cookie{
 		Name:     tokenName,
 		Value:    string(tokenString),
+		Expires:  time.Now().Add(365 * 30 * 24 * time.Hour),
```

Comment:
This timespan may not align with a remote provider's token lifespan. I wonder if we'll want to specify the expiration date here or have it sourced from the Remote Provider. // @utkarsh-pro

Reply:
@yagarwal1307 we'll want to set the timespan to something closer to 7 days or so.

Reply:
There's a bit more thought to put into this feedback. Access tokens, refresh tokens...

Reply:
Sure @leecalcote  will try to figure this out!